src/state.rs

```diff
@@ -51,6 +51,55 @@ impl State {
     Ok(())
   }
 
+  pub(crate) fn load(&mut self, path: PathBuf) -> Result<()> {
+    match path.extension() {
+      Some(ext) if ext == "txt" => {
```

Comment:
I think we should only support loading what the `image` crate supports. If we support loading text files, then we have to think about keeping the format stable, and essentially write a custom parser for it, but if our position is that it's a display and debug format only, we can change it whenever we want. And I think we'll want to, i.e. using [special characters](https://en.wikipedia.org/wiki/Box-drawing_character) to make it prettier.

Reply:
Agreed. Makes things much simpler.